docker/bash.sh

```diff
@@ -154,6 +162,7 @@ ${DOCKER_BINARY} run --rm --pid=host\
     ${WORKSPACE_VOLUMES}\
     -v ${WORKSPACE}:/workspace \
     -v ${SCRIPT_DIR}:/docker \
+    -v ${CI_DOCKER_MOUNT_DIR}:${CI_DOCKER_MOUNT_DIR} \
```

Comment:
i think you need to handle the case where --mount is not given

Reply:
thanks for pointing out. I changed the command to be empty when --mount is not available.